core/start-blueos-core

```diff
@@ -85,6 +86,12 @@ else
     echo "Waring: Using dockers's resolv.conf, this may cause DNS issues if the host's network configuration changes."
 fi
 
+# fix permissions for logging folders:
+find /var/logs/blueos -type d -exec chmod 777 {} \;
```

Comment:
The default folder permission should be "755", no ?

Reply:
yep, updated